frontend/src/metabase/dashboard/components/DashCard/DashCardMenu/DashCardMenu.tsx

```diff
@@ -165,12 +177,11 @@ DashCardMenu.shouldRender = ({
     question.datasetQuery(),
   );
 
-  if (isEmbed) {
```

Comment:
is this prop not passed anymore? By the way I noticed `isPublicOrEmbedded` was added only 2 weeks ago, so I assume it should have replaced this if block in the first place?

Reply:
Ok that makes sense.